MediaGalleryUi/Controller/Adminhtml/Image/Delete.php

```diff
@@ -99,14 +99,15 @@ public function execute()
         }
 
         try {
-            $assets = $this->getAssetsByIds->execute([$imageId]);
-            /** @var AssetInterface $asset */
-            $asset = current($assets);
-            $this->deleteImage->execute($asset);
+            $this->deleteImage->execute($this->getAssetsByIds->execute($imageIds));
             $responseCode = self::HTTP_OK;
+            $prefix = count($imageIds) > 1 ? 'images' : 'image';
             $responseContent = [
                 'success' => true,
-                'message' => __('You have successfully removed the image "%image"', ['image' => $asset->getTitle()]),
+                'message' => __(
+                    'You have successfully removed the "%image" ' . $prefix,
```

Comment:
Let's have the following success messages:
 - 1 asset: `The asset "%assetName" has been successfully deleted.`
 - more than 1 asset: `The assets have been successfully deleted.`